README.md

```diff
@@ -49,20 +49,15 @@ The main steps to integrate iob-eth core into an iob-soc system:
     include $(ETHERNET_DIR)/software/embedded/embedded.mk
     ```
 3. Define `SIM` for simulation:
-    1. Add `SIM` variable in `hardware/simulation/simulation.mk` before
-       including `hardware/hardware.mk`:
-    ```Make
-    SIM=1
-    DEFINE+=$(defmacro)SIM=$(SIM)
-    ```
-    2. Add `SIM` to firmware in case of simulation in
+    This allows using `#ifdef SIM` in the firmware.
```

Comment:
so the sw still uses this macro? It is best to pass mac address as a macro then sim

Reply:
The firmware needs to use this macro to know how to transfer files to/from console. This is required because file transfers with the console are not supported by ETHERNET in simulation (they must be done via UART).

Reply:
During simulation, the ETHERNET peripheral should be connected to the `iob_eth_tb_gen` loopback module in the simulation top system (or testbench). Currently, it has no way of communicating with the python console.